### DISCUSSION TITLE
When running the WebUI in chat mode I get some unwanted extra ### Human followed by ### Assistant dialogue

### DISCUSSION THREAD

**Discussion by nickmincu** [oobabooga/text-generation-webui#786] [Q&A]
how do I setup the configuration to rif of the extra ### Human - ### Assistant Dialogue?
also very often the human-assistant dialog is not related to the question at all. sometimes it's related to previous questions from the current context.

![image](https://user-images.githubusercontent.com/119722240/229974919-d8119a0b-4cdf-4565-a49f-d6acc15d04f3.png)

**Comment by mcmonkey4eva** [comment]
See my answer @ https://github.com/oobabooga/text-generation-webui/discussions/839#discussioncomment-5557486
